src/main/java/com/elytradev/concrete/inventory/gui/ConcreteContainer.java

```diff
@@ -197,7 +190,7 @@ public ItemStack transferToInventory(ItemStack stack, IInventory inventory) {
 					ItemStack dest = s.getStack();
 					
 					//If the two items can stack together and the existing stack can hold more items...
-					if (canStackTogether(result, dest) && dest.getCount() < s.getItemStackLimit(dest)) {
+					if (ItemHandlerHelper.canItemStacksStack(result, dest) && dest.getCount() < s.getItemStackLimit(dest)) {
```

Comment:
* [x] Returns false if both stacks are empty
* [x] Compares their NBT's
* [ ] Checks `dest.isStackable()`
* [x] Compares their items
* [x] Compares their damage/metadata
* [x] Compares their capabilities

Currently adding a helper `getMaxStackSize` function to replace `s.getItemStackLimit(dest)`, which will address the one unchecked item.